webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/domain/policies/DirectiveDiff.scala

```diff
@@ -75,22 +75,47 @@ final case class ModifyDirectiveDiff(
     id:            DirectiveId,
     name:          String, // keep the name around to be able to display it as it was at that time
 
-    modName:             Option[SimpleDiff[String]] = None,
-    modTechniqueVersion: Option[SimpleDiff[TechniqueVersion]] = None,
-    modParameters:       Option[SimpleDiff[SectionVal]] = None,
-    modShortDescription: Option[SimpleDiff[String]] = None,
-    modLongDescription:  Option[SimpleDiff[String]] = None,
-    modPriority:         Option[SimpleDiff[Int]] = None,
-    modIsActivated:      Option[SimpleDiff[Boolean]] = None,
-    modIsSystem:         Option[SimpleDiff[Boolean]] = None,
-    modPolicyMode:       Option[SimpleDiff[Option[PolicyMode]]] = None,
-    modTags:             Option[SimpleDiff[Set[Tag]]] = None
+    modName:             Option[SimpleDiff[String]],
+    modTechniqueVersion: Option[SimpleDiff[TechniqueVersion]],
+    modParameters:       Option[SimpleDiff[SectionVal]],
+    modShortDescription: Option[SimpleDiff[String]],
+    modLongDescription:  Option[SimpleDiff[String]],
+    modPriority:         Option[SimpleDiff[Int]],
+    modIsActivated:      Option[SimpleDiff[Boolean]],
+    modIsSystem:         Option[SimpleDiff[Boolean]],
+    modPolicyMode:       Option[SimpleDiff[Option[PolicyMode]]],
+    modTags:             Option[SimpleDiff[Tags]]
```

Comment:
For information: when you do that kind of change, you need to ensure that it does not impact a plugin (here, the one that could be impacted is `change-validation`). 
(and it's ok, it is not, it's just so that you won't be surprise the first time a change in rudder breaks a plugin, and so you will understand why our plugin build system is so complex)